src/Dropdown/Dropdown.js

```diff
@@ -147,67 +158,204 @@ export default class Dropdown extends Component<Props, State> {
     return <Root {...rootProps}>{children}</Root>;
   }
 
-  getTriggerProps = (props: Object) => {
-    const contentId = `${this.id}-dropdownContent`;
-    const { isOpen } = props;
-
-    return {
-      ...props,
-      'aria-activedescendant': isOpen
-        ? this.selectedItemId || `${contentId}-menu`
-        : undefined,
-      'aria-haspopup': true,
-      contentId,
-      onKeyDown: this.onTriggerKeyDown
-    };
+  getContentId = () => {
```

Comment:
Alphabetize methods?

Reply:
Maybe, but prolly not right now, unless everyone feels very strongly about it.  The methods are currently in a logical order wherein similar/related methods are grouped together.  During development, I found it much easier to work with as it avoided lots of jumping around.